tests/components/matter/test_binary_sensor.py

```diff
@@ -31,7 +31,7 @@ async def test_contact_sensor(
     """Test contact sensor."""
     state = hass.states.get("binary_sensor.mock_contact_sensor_contact")
     assert state
-    assert state.state == "on"
+    assert state.state == "off"
```

Comment:
Why does these tests change state?

Reply:
because it was reversed before and I fixed it :-)

Reply:
Ok.